go-controller/pkg/ovn/ovn.go

```diff
@@ -838,17 +838,26 @@ func (oc *Controller) aclLoggingCanEnable(annotation string, nsInfo *namespaceIn
 	if err != nil {
 		return false
 	}
+
+	// Using newDenyLoggingLevel and newAllowLoggingLevel allows resetting nsinfo state.
+	// This is important if a user sets either the allow level or the deny level flag to an
+	// invalid value or after they remove either the allow or the deny annotation.
+	// If either of the 2 (allow or deny logging level) is set with a valid level, return true.
+	newDenyLoggingLevel := ""
+	newAllowLoggingLevel := ""
 	okCnt := 0
 	for _, s := range []string{"alert", "warning", "notice", "info", "debug"} {
 		if aclLevels.Deny != "" && s == aclLevels.Deny {
-			nsInfo.aclLogging.Deny = aclLevels.Deny
+			newDenyLoggingLevel = aclLevels.Deny
 			okCnt++
 		}
 		if aclLevels.Allow != "" && s == aclLevels.Allow {
-			nsInfo.aclLogging.Allow = aclLevels.Allow
+			newAllowLoggingLevel = aclLevels.Allow
 			okCnt++
 		}
 	}
```

Comment:
Not your change but does `aclLevels.Deny != "" && s == aclLevels.Deny` make sense to you or is it me not understanding? Would it be enough to just `s == aclLevels.Deny`?

Reply:
Good catch, done